index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Mint Signup Form</title>
+        <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+        <link rel="stylesheet" href="styles/style.css">
+    </head>
+    <body>
+        <div class="top">
+            <div class="top-text">
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <div class="signin-btn">
+                <a href="#">
+                    <button type="button">Sign In</button>
```

Comment:
A button cannot be a descendant of an `a` tag.

Use either a` button` tag or an `a` tag but not one inside another. 

If you are looking for the hover effect for the button you can do so, for now, with `cursor: pointer` CSS property value pair for your button or you can style the anchor tag to be like a button.

Hope this helps. 😄

Reply:
Thanks Maya for your review.
The a tag has been removed.